Automattic/vip-go-mu-plugins
# PHPUnit: Add "name" attribute for "testsuite" element to avoid warnings

## Description


When running `phpunit ./tests/test-a8c-files-utils.php` or `phpunit ./tests/test-a8c-files-utils.php  -c phpunit-multisite.xml`  in my local machine, I get this warning: 

>   Warning - The configuration file did not pass validation!
>   The following problems have been detected:
> 
>   Line 14:
>   - Element 'testsuite': The attribute 'name' is required but missing.
> 
>   Test results may not be as expected.

This PR simply adds the `name` attribute to PHPUnit config files to clear up this warning. 
Reference: https://phpunit.readthedocs.io/en/7.5/configuration.html#the-testsuite-element

## Changelog Description

### PHPUnit: Add "name" attribute for "testsuite" element to avoid warnings

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Run `phpunit ./tests/test-a8c-files-utils.php` or `phpunit ./tests/test-a8c-files-utils.php  -c phpunit-multisite.xml`. I chose a specific file for testings to reduce the time for verifying this issue. 
2. No longer see any warning.
